bin/cron/deploy_to_test

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env ruby
+
+require 'cdo/chat_client'
+require 'cdo/github'
+require 'cdo/only_one'
+
+# Checks whether the topic of Slack#developers includes "DTT: yes", updating it
+# to reflect auto-DTT if so.
+# @return [Boolean] Whether to proceed with the auto-DTT.
+def check_and_update_developers_topic
+  current_topic = Slack.get_topic('developers')
+  unless current_topic.include? 'DTT: yes'
+    ChatClient.message(
+      'infra-test',
+      "auto-DTT aborted (\#developers topic is #{current_topic}).",
+      color: 'yellow'
+    )
+    return false
+  end
+
+  new_topic = current_topic.gsub('DTT: yes', 'DTT: no (robo-DTT in progress)')
+  Slack.update_topic(new_topic)
+
+  # TODO(asher): Determine if we should continue with the DTT if the Slack topic
+  # update failed.
```

Comment:
TODO should probably be addressed before merge.  Does `Slack.update_topic` raise when it fails?